agbenchmark/tests/regression/regression_tests.json

```diff
@@ -1 +0,0 @@
-{}
```

Comment:
should be put outside of the agbenchmark folder because it's defined for each client